services/transactionService.ts

```diff
@@ -24,7 +24,7 @@ export async function getTransactionInfoFromNetwork(
   const userTransactionsCount = await web3.eth.getTransactionCount(
     input.fromAddress,
   );
-  if (nonce && userTransactionsCount <= nonce) {
+  if ( nonce !== undefined && userTransactionsCount <= nonce) {
```

Comment:
It can be null !! null is different than undefined
```suggestion
  if ( typeof nonce !== 'number' && userTransactionsCount <= nonce) {
```
can be a solution

Reply:
@aminlatifi  Yeah you are right, it's better